sources/packages/api/src/services/restriction/student-restriction.service.ts

```diff
@@ -93,9 +96,11 @@ export class StudentRestrictionService extends RecordDataModelService<StudentRes
       ])
       .innerJoin("studentRestrictions.restriction", "restriction")
       .innerJoin("studentRestrictions.student", "student")
-      .where("student.id = :studentId", { studentId })
-      .orderBy("studentRestrictions.isActive", "DESC")
-      .getMany();
+      .where("student.id = :studentId", { studentId });
+    if (onlyActive) {
+      query.andWhere("studentRestrictions.isActive = true");
+    }
+    return query.orderBy("studentRestrictions.isActive", "DESC").getMany();
```

Comment:
I believe that we should include also some dates like `createdAt` in the order by.